### ISSUE TITLE
"Stream accepts 0 type arguments" error

### ISSUE THREAD

**Issue by johnpryan** [johnpryan/flutter_map#154]
```
cache/hosted/pub.dartlang.org/flutter_map-0.2.0/lib/src/map/flutter_map_state.dart:34:3: Error: 'Stream' expects 0 type arguments.
compiler message:   Stream<Null> _merge(LayerOptions options) {
```

**Comment by ibrierley** [comment]
Can you include the code you have to use it.

**Comment by kengu** [comment]
The problem stems from the fact that flutter_map assumes all streams to be streams of `null` values. We could relax this by changing from `Stream<Null>` to `Stream<dynamic>`?

**Comment by johnpryan** [comment]
I think the [recommended way](https://medium.com/dartlang/dart-2-legacy-of-the-void-e7afb5f44df0) is to use `void`

**Comment by kengu** [comment]
@johnpryan, I agree, if the stream should not carry any information. However, I could see use cases where the `option. stream` parameter could be used to notify a plugin that some state has changed, where actual state is consumed by the plugin from the stream directly. This allow for a reactive programming model without any additional custom code for pulling or pushing the state to the plugin.

**Comment by johnpryan** [comment]
To fix this and #164 will require some more information and solid steps to reproduce. AFAICT this is not affecting all users right now.

**Comment by johnpryan** [comment]
this issue seems stale, so I'm going to close it.
